src/store/search.js

```diff
@@ -178,7 +87,7 @@ export const state = () => ({
     size: '',
     source: '',
     searchBy: '',
-    mature: false,
+    mature: '',
```

Comment:
I think it would mean that we would have to move the custom logic somewhere else. Because there would have to be the logic of removing 'false' from the API query. Or do we want to send 'mature=false' with every query, instead of only adding 'mature=true' to the queries that set mature to true?

Reply:
ah yes, that's a good point. But would using '' and 'true' fix that?

Reply:
In search-query-transform we use the `query` object from the search store, and convert it into the the object that is used to create the API request URL. There, we use the `hideEmpty` parameter set to true, so any query parameter that is equal to `''` is discarded, and non-blank parameters are used in the query string.

Reply:
So, in short, `mature: 'anykindofstring'` is converted to `/?q=cat&mature=anykindofstring`, but `mature: ''` is converted to `?q=cat`.

Reply:
Yes, that's what I thought, thanks! So we _could_ use '' for the default, _off_ value of the mature filter, and the string 'true' as the value for _showing_ mature images, which would allow us to avoid any custom conditionals for the mature filter in our search query transformation logic